sktime/classification/deep_learning/rnn.py

```diff
@@ -80,6 +93,7 @@ def __init__(
         self.optimizer = optimizer
         self.history = None
         self._network = RNNNetwork(random_state=random_state, units=units)
+        self.n_epochs = n_epochs
```

Comment:
This is overriding L82 if I'm not wrong. If user passes `num_epochs = 50`, the rest of the code should use `50` as that's what user wants and IMO we should respect that till actual deprecation.

@fkiraly will it make sense to set self.n_epochs to num_epochs if it's not None?

Reply:
So should I remove line 96 from code, or put self.n_epochs = n_epochs in the else block after line 81

Reply:
> @fkiraly will it make sense to set self.n_epochs to num_epochs if it's not None?

That will violate the `sklearn` interface assumptio, any param of `__init__` should be written to `self` and never changed. I would suggest using an auxiliary `self._n_epochs` or similar